Services/UI/OJS.Services.Ui.Business/Validation/Implementations/ContestValidationService.cs

```diff
@@ -16,18 +15,14 @@ public ValidationResult GetValidationResult((Contest?, string, bool, bool) item)
 
         if (contest == null ||
             contest.IsDeleted ||
-            (!contest.IsVisible && !isUserLecturerInContest))
+            (!contest.IsVisible && !isUserLecturerInContest && !isUserAdmin))
         {
-            return ValidationResult.Invalid(
-                ValidationMessages.Contest.NotFound,
-                ContestValidation.ContestIsFound.ToString());
+            return ValidationResult.Invalid(string.Format(ValidationMessages.Contest.NotFound, contest?.Name));
```

Comment:
If contest is null then then contest?.Name will be empty
Id can be returned with message instead of empty string

Reply:
Done.